app/src/main/res/layout/activity_forgot_password.xml

```diff
@@ -75,7 +75,7 @@
         android:layout_width="250dp"
         android:layout_height="wrap_content"
         android:layout_marginTop="@dimen/margin_medium"
-        android:background="@color/md_grey_300"
+        android:background="@drawable/rounded_button"
```

Comment:
No, Should i do that? it looks like this.

![screen shot 2017-07-03 at 2 18 07 pm](https://user-images.githubusercontent.com/21264401/27785211-811873c8-5ffa-11e7-9e63-271d33064c13.png)